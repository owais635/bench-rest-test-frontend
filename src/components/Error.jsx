import React from "react";
import styled from "styled-components";

const Container = styled.div`
  display: flex;
  justify-content: center;
  height: 40vh;
  align-items: center;
  flex-direction: column;
`;

export default function Loading() {
  return (
    <Container>
      <h2>Sorry. Something went wrong.</h2>
      <h3>Please try again later.</h3>
    </Container>
  );
}
