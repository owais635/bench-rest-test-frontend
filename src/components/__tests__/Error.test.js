import React from "react";
import renderer from "react-test-renderer";
import Error from "../Error";

describe("Error", () => {
  test("should render", () => {
    const component = renderer.create(<Error />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
