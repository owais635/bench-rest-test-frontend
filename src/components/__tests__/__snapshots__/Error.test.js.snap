// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render 1`] = `
<div
  className="sc-bdvvtL gAVdrl"
>
  <h2>
    Sorry. Something went wrong.
  </h2>
  <h3>
    Please try again later.
  </h3>
</div>
`;
